app/helpers/dashboards_helper.rb

```diff
@@ -37,4 +37,10 @@ def call_glyphicon
       tag(:span, class: ['sr-only'], text: 'Call')
     ]
   end
+
+  def start_day
```

Comment:
SUGGESTION: This is virtually identical to `def self.start_of_week`. I would see if you could abstract it out into a helper somewhere and just call that rather than have two methods that are largely the same floating around.